static/images/cards/ic-api.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="94" height="94" viewBox="0 0 94 94">
```

Comment:
Do you need all the svg images there?

Reply:
Thank for the comment! You are right, I deleted no-important files.